ui/src/app/kvm/components/CPUColumn/CPUPopover/CPUPopover.tsx

```diff
@@ -36,18 +33,16 @@ const CPUPopover = ({
           <div className="cpu-popover__header p-table__header">CPU cores</div>
           <div className="cpu-popover__primary">
             <div className="u-align--right" data-test="allocated">
-              {allocated}
+              {coresWithOver.allocated_tracked}
             </div>
             <div className="u-vertically-center">
               <i className="p-circle--link"></i>
             </div>
-            <div data-test="allocated-label">
-              {showOther ? "Project" : "Allocated"}
-            </div>
+            <div>Allocated</div>
```

Comment:
I decided to remove the change in language here so it's consistent all throughout the KVM pages